stetho/src/main/java/com/facebook/stetho/inspector/database/SqliteDatabaseDriver.java

```diff
@@ -82,9 +80,20 @@ private void bootstrapNewPeer(JsonRpcPeer peer) {
       eventParams.database = databaseParams;
 
       peer.invokeMethod("Database.addDatabase", eventParams, null /* callback */);
```

Comment:
I'd really prefer not to repeat this method.  Can we refine the driver implementation to something more database-focused and use a generalized harness to interact with Chrome DevTools:

```java
interface DatabaseDriver {
  List<String> getDatabaseNames();
  List<String> getTableNames(String databaseName);
  ExecuteSQLResponse executeSQL(String database, String query, ExecuteResultHandler<ExecuteSQLResponse>);
}
```

We may even wish to generalize `ExecuteResultHandler` a bit so that it's not so directly cursor-focused but rather works more closely as an abstraction of how Chrome DevTools interprets the response.